avro-format.md

```diff
@@ -47,108 +48,81 @@ The CloudEvents type system MUST be mapped to Avro types as follows.
 | String        | [string][avro-primitives]                                              |
 | Binary        | [bytes][avro-primitives]                                               |
 | URI           | [string][avro-primitives] following [RFC 3986 §4.3][rfc3986-section43] |
 | URI-reference | [string][avro-primitives] following [RFC 3986 §4.1][rfc3986-section41] |
 | Timestamp     | [string][avro-primitives] following [RFC 3339][rfc3339] (ISO 8601)     |
-| Any           | See [2.2](#22-mapping-any-typed-attributes)                            |
 
 Extension specifications MAY define diverging mapping rules for the values of
 attributes they define.
 
-### 2.2 Mapping Any-typed Attributes
-
-`Any`-typed CloudEvents values can either hold a `String`, or a `Binary` value,
-or a `Map`, or any of all other types. Avro type system satisfies this requirement by employing a recursive reference,
-where a `record` type is referenced as a value inside of its own `map`.
-
-Example:
-
-```json
-{
-  "type":"record",
-  "name":"MyRecord",
-  "fields":[
-    {
-      "name":"wow",
-      "type":{
-        "type":"map",
-        "values":[
-          "null",
-          "string",
-          "MyRecord"
-        ]
-      }
-    }
-  ]
-}
-```
-
 ### 2.3 OPTIONAL Attributes
 
-CloudEvents Spec defines OPTIONAL attributes. The Avro format defines that
-these fields MUST use the `null` type and the actual type through
-the [union][avro-unions].
+CloudEvents Spec defines OPTIONAL attributes. The Avro format defines that these
+fields MUST use the `null` type and the actual type through the
+[union][avro-unions].
 
 Example:
 
 ```json
-[
-  "null",
-  "string"
-]
+["null", "string"]
 ```
 
 ### 2.4 Definition
 
-Users of Avro MUST use a message whose binary encoding is identical
-to the one described by the [CloudEvent Avro Schema](./spec.avsc):
+Users of Avro MUST use a message whose binary encoding is identical to the one
+described by the [CloudEvent Avro Schema](./spec.avsc):
 
 ```json
 {
-  "namespace":"io.cloudevents",
-  "type":"record",
-  "name":"CloudEvent",
-  "version":"0.4-wip",
-  "doc":"Avro Event Format for CloudEvents",
-  "fields":[
+  "namespace": "io.cloudevents",
+  "type": "record",
+  "name": "CloudEvent",
+  "version": "0.4-wip",
+  "doc": "Avro Event Format for CloudEvents",
+  "fields": [
     {
-      "name":"attribute",
-      "type":{
-        "type":"map",
-        "values":[
-          "null",
-          "int",
-          "string",
-          "bytes",
-          "CloudEvent"
-        ]
+      "name": "attribute",
+      "type": {
+        "type": "map",
+        "values": ["null", "boolean", "int", "string", "bytes"]
       }
+    },
+    {
+      "name": "data",
+      "type": "bytes"
     }
   ]
 }
 ```
 
-## 3 Examples
+## 3 Data
+
+The `data` of the CloudEvent should be encoded in a top-level field called
+`data` of type `bytes`. No additional encoding should be done, regardless of the
```

Comment:
s/should/will/. 
Or if you want it to be normative; Additional encoding MUST NOT be done, regardless ....